internal/test_helpers/scenarios/gleam/README.md

```
@@ -0,0 +1,65 @@
+1. Commit your changes and run `git push origin master` to submit your solution
+   to CodeCrafters. Test output will be streamed to your terminal.
+
+{{#language_is_python}}
+
+# Troubleshooting
+
+## module `socket` has no attribute `create_server`
+
+When running your server locally, you might see an error like this:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)